src/indicators/design-guidelines.md

```diff
@@ -0,0 +1,22 @@
+The following is a brief set of design guidelines for indicators:
+
+## Calculators
+
+ - calculators should operate on an input array of data, transforming it into a new array containing the indicator output values.
```

Comment:
Very pedantic, but some of the list items start with an uppercase letter and some with a lowercase letter. Doesn't really matter much though! :-)